packages/calcite-components/conventions/Styling.md

```diff
@@ -32,7 +32,9 @@ Add a class to handle the logic in the component class.
 
 ## Light Mode/Dark Mode
 
-In the [global CSS file](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/assets/styles/global.scss), we specify the values of each color for both light and dark mode. This enables theming to be inherited throughout a component tree. Consider this valid example:
+Light and Dark modes are now provided via Calcite Design Tokens.
```

Comment:
>
> Light and dark modes are now provided via Calcite Design Tokens. Color context modes are important to end-users because it may help to reduce eye-strain and save power. A light or dark mode can also be set explicitly to meet team project requirements.
>